install_files/ansible-base/roles/reset-ssh-key/tasks/reset-ssh-key.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Set authorized key, removing all the authorized keys already set
```

Comment:
The "removing all the authorized keys" bit may sound a bit scary in command output, perhaps shorten to something like "Set authorized key exclusively to the public key configured on this workstation"

Reply:
Changed